src/lib/contact/contactClasses/manageContact.ts

```diff
@@ -0,0 +1,85 @@
+import type {
```

Comment:
you can leave this "manage" part in `contact.ts` imo, or if you don't rename the `Contact` class to `BaseContact` to avoid confusion :smile: